# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1219772294 14400
#      Tue Aug 26 13:38:14 2008 -0400
# Node ID e04a4beeb28316f09a61953d329ddfc38dc82609
# Parent  f819e89923670e5eddf7c1626eaed4ce05b718bb
graphics.cc (figure::properties::get_title): avoid gcc warning

diff --git a/src/graphics.cc b/src/graphics.cc
--- a/src/graphics.cc
+++ b/src/graphics.cc
@@ -1944,20 +1944,20 @@ figure::properties::set_position (const 
 }
 
 std::string
 figure::properties::get_title (void) const
 {
   if (is_numbertitle ())
     {
       std::ostringstream os;
-      std::string name = get_name ();
+      std::string nm = get_name ();
 
       os << "Figure " << __myhandle__.value ();
-      if (! name.empty ())
+      if (! nm.empty ())
 	os << ": " << get_name ();
 
       return os.str ();
     }
   else
     return get_name ();
 }
 
